docs/content/latest/develop/gitdev/gitpod.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Gitpod
+linkTitle: Gitpod
+description: Gitpod integrated development environment
+menu:
+  latest:
+    identifier: gitpod
+    parent: gitdev
+    weight: 592
+isTocNested: true
+showAsideToc: true
+---
+
+Use [Gitpod](https://www.gitpod.io) workspaces to provision an instant development environment with a pre-configured YugabyteDB.
+
+Gitpod is a configurable ready-to-code cloud development environment accessible via a browser. A Gitpod workspace includes everything you need to develop for a specific repository, including the Visual Studio Code editing experience, common languages, tools, and utilities. Instantly this sets up a cloud-hosted, containerized, and customizable editing environment.
+
+Follow the steps on this page to set up a Gitpod workspace environment with a pre-configured YugabyteDB. For details on Gitpod workspaces, refer to the [Gitpod documentation](https://www.gitpod.io/docs/).
+
+## Requirements
+
+Gitpod doesn't require anything in your local workstation other than a code editor and Git CLI. Much of the development happens in the cloud through a web browser.
+
+## Get started with a boot app
+
+You can find the source at [Spring Boot todo on GitHub](https://github.com/yugabyte/yb-todo-app.git). 
+
+The easy way to get started with Gitpod is to simply fork this [source repo](https://github.com/yugabyte/yb-todo-app.git) and initialize the Gitpod workspace environment by invoking `https://gitpod.io/#[REPO_URL]` in a browser window. Replace `[REPO_URL]` with your forked repository URL and you should already have been conneted to the Gitpod account before launching the browser URL.
```

Comment:
"...should already have been connected..." - does this mean I should be logged into my Gitpod account before I do this step?

Reply:
Also, if i'm doing the easy way, do i need to follow any of the instructions on this page other than "fork the repo, browse to gitpod.io/#repo_url"? I suspect we need to say "do this, then skip to [Customize the Gitpod environment]"

Reply:
> "...should already have been connected..." - does this mean I should be logged into my Gitpod account before I do this step?
Yes, u need to be logged in. If not, you would be taken through the login flow.


> Also, if i'm doing the easy way, do i need to follow any of the instructions on this page other than "fork the repo, browse to gitpod.io/#repo_url"? I suspect we need to say "do this, then skip to [Customize the Gitpod environment]"
Easy way => just fork and launch the URL